models/Tree.php

```diff
@@ -294,7 +297,7 @@ public function removeNode($softDelete = true, $currNode = true)
                     }
                 }
             }
-            if (!empty($this->nodeRemovalErrors)) {
+            if ($this->nodeRemovalErrors) {
```

Comment:
`nodeActivationErrors` and `nodeRemovalErrors` is an array even if it is initialized ... and checking if it is not `empty` is ok.